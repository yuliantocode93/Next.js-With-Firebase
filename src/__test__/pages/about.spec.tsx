import AboutPage from "@/pages/about";
import { render, screen } from "@testing-library/react";

describe("About Page", () => {
  it("should render the About Page", () => {
    const { asFragment } = render(<AboutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
