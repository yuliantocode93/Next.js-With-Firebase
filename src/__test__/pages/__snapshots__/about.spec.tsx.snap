// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should render the About Page 1`] = `
<DocumentFragment>
  <div>
    <h1
      id="title"
    >
      About Page
    </h1>
  </div>
</DocumentFragment>
`;
